### ISSUE TITLE
Select stuck in animation after clicking outside

### ISSUE THREAD

**Issue by jacobfederer** [trimox/angular-mdc-web#1520]
### What Angular MDC version are you using?

41

### What browser(s) is this bug affecting?

Safari

### What OS are you using?

MacOS

### What are the steps to reproduce the bug?

Here is my code:

```
  <mdc-select class="margin-left" label="Kundengruppe" (selectionChange)="onChangeKundenFilter($event.value)" autosize
              #kundenGruppenFilter>
    <option [value]="..." selected>...</option>
    <option *ngFor="let group of groups" [value]="group.getId()">{{group.getName()}}</option>
    <option [value]="...">Alle</option>
  </mdc-select>
```

### What is the expected behavior?

That after clicking outside of the select the select returns to the default resting state.

### What is the actual behavior?

Upon clicking the select: 

![bildschirmfoto 2018-11-02 um 17 32 08](https://user-images.githubusercontent.com/1516212/47910378-43aaf100-dec5-11e8-809c-d0c83a7528de.png)

After clicking outside the select: 

![bildschirmfoto 2018-11-02 um 17 32 12](https://user-images.githubusercontent.com/1516212/47910401-5291a380-dec5-11e8-8797-f49e05cbe067.png)

**Comment by trimox** [comment]
Just noticed the leading icon, nice! I need to add an example for demos.

**Comment by trimox** [comment]
Resolved as part of #1402
